code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -44,12 +48,16 @@
 	infection = C.getorganslot("zombie_infection")
 	if(!infection)
 		infection = new(C)
+	stunmod = 0.7
+	C.status_flags += IGNORESLOWDOWN

```

Comment:
This is not how you update a binary flag, same goes for the removal in on_species_loss

https://tgstation13.org/wiki/Binary_flags

please make sure you use the flags correctly, some of them are lists and some of them are binary flags and this is confusing but a quick github search for the particular var will quickly show you which type it is.